[tcat] fix missing Python modules in tcat-ble-client pyproject.toml (#11984)
openthread/openthread: tools/tcat_ble_client/README.md

# TCAT Commissioner (BBTC) Client

## Overview

This is a Python implementation of a Bluetooth-Based Thread Commissioning (BBTC) client, based on Thread's TCAT (Thread Commissioning over Authenticated TLS) functionality.

## Installation

If you don't have the poetry module installed (check with `poetry --version`), install it first following the [official installation instructions](https://python-poetry.org/docs/#installation).

For example, if pipx is available:

```bash
pipx install poetry
```

If pipx is not available, it can be installed for Linux/Windows/MacOS following the [pipx installation instructions](https://pipx.pypa.io/stable/installation/).

Then, install this project using Poetry:

```
poetry install --no-root
```

This will install all the required modules to a virtual environment, which can be used by calling `poetry run <COMMAND>` from the project directory.

> Note: installation in Windows will require the [Build Tools for Visual Studio C/C++](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022) to be installed.

Reviewer: ```suggestion
Note: Installation on Windows requires that [Build Tools for Visual Studio C/C++](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022) be installed first.
```
